src/network/networkd-manager.c

```diff
@@ -599,7 +599,9 @@ int manager_new(Manager **ret, bool test_mode) {
                 .manage_foreign_nexthops = true,
                 .ethtool_fd = -EBADF,
                 .persistent_storage_fd = persistent_storage_open(),
+                .dhcp_use_domains = DHCP_USE_DOMAINS_NO,
                 .dhcp_duid.type = DUID_TYPE_EN,
+                .dhcp6_use_domains = DHCP_USE_DOMAINS_NO,
```

Comment:
`DHCP_USE_DOMAINS_NO` is zero, thus, these lines are not necessary. Please drop them.